### PR TITLE
BaSP-MR-137: Form for activities and subscriptions

### PR DESCRIPTION
**Pull request by JuanManuel-Blois**
Worked on the form for activities and subscriptions

### CODE REVIEW

File: src/Views/Admin/Activities/FormActivity/index.js [hunk 1]
```diff
@@ -32,11 +30,47 @@ const FormActivities = () => {
...
+    name: Joi.string()
+      .min(3)
+      .max(15)
+      .regex(/^[a-zA-Z]+(?: [a-zA-Z]+)*$/)
+      .required()
+      .messages({
+        'string.pattern.base': 'Activity name must contain letters only',
+        'string.min': 'Activity name cant be shorter than 3 characters',
+        'string.max': 'Activity  name cant be longer than 15 characters',
+        'string.empty': 'Activity name cant be empty'
```
Comment (LucianoStradiot) [inline src/Views/Admin/Activities/FormActivity/index.js]: Change cant for can´t

**Review by LucianoStradiot — APPROVED** [review]
LG

**Comment by LucianoStradiot** [comment]
[[FEATURE] - Form (Activities, Subscriptions)](https://trello.com/c/uzzesmB3/137-feature-form-activities-subscriptions)

**Comment by LautaroVercellini** [comment]
Looks Good!

**Pull request merged**